packages/avs-bot/src/avs.ts

```diff
@@ -0,0 +1,195 @@
+/* eslint-disable no-console */
+import axios from 'axios';
+import {CallMessage} from '@wireapp/core/dist/conversation/message/OtrMessage';
+import {CallingContent} from '@wireapp/core/dist/conversation/content';
+import {ENV as AVS_ENV, getAvsInstance, Wcall, LOG_LEVEL, REASON, CALL_TYPE} from '@wireapp/avs';
+import type {Account} from '@wireapp/core';
+import {RTCAudioSourceWrapper} from './rtcAudioSource';
+
+interface Call {
+  clientId: string;
+  conversationId: string;
+  hasVideo: number;
+  shouldRing: number;
+  timestamp: number;
+  userId: string;
+}
+
+export class Avs {
+  wCall?: Wcall;
+  wUser?: number;
+  activeCalls: Record<string, Call> = {};
+  mediaStream: MediaStream = new MediaStream();
+  audioSource: RTCAudioSourceWrapper = new RTCAudioSourceWrapper();
+
+  constructor(private readonly account: Account) {}
+
+  async initAvs(selfUserId: string, clientId: string): Promise<{wCall: Wcall; wUser: number}> {
+    const callingInstance = await getAvsInstance();
+    const wCall = this.configureCallingApi(callingInstance);
+    const wUser = this.createWUser(wCall, selfUserId, clientId);
+    this.wUser = wUser;
+    this.wCall = wCall;
+    return {wCall: wCall, wUser};
```

Comment:
I think there's no need to expose wCall and wUser to the outside

Reply:
https://github.com/wireapp/wire-web-packages/pull/3070/commits/16edff26574f9ab4f1e8b56685ea1dabf52dedd7